src/org/mozilla/javascript/regexp/NativeRegExp.java

```diff
@@ -2783,6 +2814,28 @@ protected int findPrototypeId(String s) {
         return id;
     }
 
+    private Object js_getFlags() {
```

Comment:
Is there any particular reason why these new methods are all prefixed with `js_`?

Reply:
We used this pattern at other places also to mark methods providing the impl of a specific js method/accessor call

Reply:
Yes maybe if all the other thinks are solved ;-)

Reply:
https://github.com/mozilla/rhino/wiki/DRAFT:-Stuff-you'd-want-to-know-about-the-codebase